lib/time/time.go

```diff
@@ -348,7 +348,7 @@ func (t Time) Hash() (uint32, error) {
 
 // Truth returns the truth value of an object required by starlark.Value
 // interface.
-func (t Time) Truth() starlark.Bool { return starlark.Bool(time.Time(t).IsZero()) }
```

Comment:
This was clearly just a mistake that no-one noticed until now. Thanks for the fix!

Reply:
No problem!

I'm pretty interested in this project - do you mind if I pick up a few of the TODOs? Got any new features or library modules you've been thinking about?

Reply:
> I'm pretty interested in this project - do you mind if I pick up a few of the TODOs? Got any new features or library modules you've been thinking about?

Please do! I don't have any specific new features in mind. Much of the effort of this project has been spent in trying to harmonize the two (now three) implementations around a common spec, and to be very judicious about new features. Feel free to have a crack at any TODO comment in the code, and don't hesitate to ask for clarification before spending significant effort. These issues too might be suitable tasks, depending on your tastes:

https://github.com/google/starlark-go/issues/356
https://github.com/google/starlark-go/issues/249
https://github.com/google/starlark-go/issues/270
https://github.com/google/starlark-go/issues/271
https://github.com/google/starlark-go/issues/292

Many thanks for the kind offer.